.travis.yml

```diff
@@ -1,42 +1,64 @@
 ---
 language: node_js
 node_js:
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
   - "6"
-  - "8"
```

Comment:
We only want to test on the minimum and not all supported versions? Why is that?

Reply:
To avoid consuming too many resources on travis.